.github/actions/run-dagger-pipeline/action.yml

```diff
@@ -89,22 +94,18 @@ runs:
       id: get-start-timestamp
       shell: bash
       run: echo "name=start-timestamp=$(date +%s)" >> $GITHUB_OUTPUT
-    - name: Install Python 3.10
-      uses: actions/setup-python@v4
-      with:
-        python-version: "3.10"
-        token: ${{ inputs.github_token }}
-    - name: Install ci-connector-ops package
+    - name: Install airbyte-ci binary
+      id: install-airbyte-ci
       shell: bash
       run: |
-        pip install pipx
-        pipx ensurepath
-        pipx install airbyte-ci/connectors/pipelines/
+        curl -sSL ${{ inputs.airbyte_ci_binary_url }} --output airbyte-ci-bin
+        sudo mv airbyte-ci-bin /usr/local/bin/airbyte-ci
+        sudo chmod +x /usr/local/bin/airbyte-ci
     - name: Run airbyte-ci
       shell: bash
       run: |
         export _EXPERIMENTAL_DAGGER_RUNNER_HOST="unix:///var/run/buildkit/buildkitd.sock"
-        airbyte-ci --is-ci --gha-workflow-run-id=${{ github.run_id }} ${{ inputs.subcommand }} ${{ inputs.options }}
+        airbyte-ci --disable-dagger-run --is-ci --gha-workflow-run-id=${{ github.run_id }} ${{ inputs.subcommand }} ${{ inputs.options }}
```

Comment:
what does `--disable-dagger-run` do?

Reply:
This is what enables the terminal UI when running locally.
We don't want this in CI because it might expose sensitive information / clutter the GHA logs.
It's called `disable dagger run` because getting the terminal UI is achieved by running `dagger run airbyte-ci` (using Dagger's CLI)